src/chainer.test.js

```diff
@@ -1 +1,31 @@
+'use strict';
+
 const { chainer } = require('./chainer');
+
+describe('chainer', () => {
+  it('should chain the unary functions correctly', () => {
```

Comment:
It would be great to add more test cases to cover different scenarios. For instance, what happens if the input array is empty, contains non-function elements, or the returned function is invoked without arguments.